packages/react-native-sdk/src/components/Call/CallLayout/CallParticipantsGrid.tsx

```diff
@@ -31,19 +35,26 @@ export const CallParticipantsGrid = ({
   ParticipantVideoFallback,
   ParticipantView,
   VideoRenderer,
+  showLocalParticipant = false,
 }: CallParticipantsGridProps) => {
   const {
     theme: { colors, callParticipantsGrid },
   } = useTheme();
-  const { useRemoteParticipants, useParticipants } = useCallStateHooks();
+  const { useRemoteParticipants, useParticipants, useLocalParticipant } =
+    useCallStateHooks();
   const _remoteParticipants = useRemoteParticipants();
   const allParticipants = useParticipants();
   const remoteParticipants = useDebouncedValue(_remoteParticipants, 300); // we debounce the remote participants to avoid unnecessary rerenders that happen when participant tracks are all subscribed simultaneously
+  const localParticipant = useLocalParticipant();
 
   const showFloatingView =
     remoteParticipants.length > 0 && remoteParticipants.length < 3;
 
-  const participants = showFloatingView ? remoteParticipants : allParticipants;
+  const participants = showFloatingView
+    ? showLocalParticipant
+      ? [localParticipant!!]
+      : remoteParticipants
```

Comment:
```suggestion
    ? showLocalParticipant && localParticipant
      ? [localParticipant]
      : remoteParticipants
```

this is safer I think.. otherwise can cause a crash by using this component individually and passing showLocalParticipant to be true with having a local participant

Reply:
Done